### PR TITLE
[13.0] connector_importer: misc improvements

### PR DESCRIPTION
**Pull request by simahawk**
~Depends / includes https://github.com/OCA/connector-interfaces/pull/72 to switch to MQT=PIP~
~You can ignore the first 2 commits.~

### CODE REVIEW

**Comment by simahawk** [comment]
@sebalix this should interest you https://github.com/OCA/connector-interfaces/pull/73/commits/bbddc9a1bf3e380ac51c2e28b300190c09cab11c

File: connector_importer/models/sources/source_consumer_mixin.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    @api.depends("source_model", "source_id")
+    def _compute_source_ref_id(self):
+        for item in self:
+            item.source_ref_id = False
+            if not item.source_id or not item.source_model:
+                continue
+            item.source_ref_id = "{0.source_model},{0.source_id}".format(item)
+
+    @api.model
+    @tools.ormcache("self")
```
Comment (lmignon) [inline connector_importer/models/sources/source_consumer_mixin.py]: ???'@tools.ormcache("self")`????

Comment (simahawk) [reply connector_importer/models/sources/source_consumer_mixin.py]: it's equivalent to not pass anything. BUT I think it's just a left over from older implementations as there's nothing to compute and cache here... I'll drop it. Thanks for having a look ;)

**Review by lmignon — APPROVED** [review]
LGTM (Code review)

**Comment by simahawk** [comment]
@sebalix would you mind passing by? I promise you'll be "rewarded" with the next PR :wink:

**Pull request merged**